modules/ROOT/pages/amazon-sqs-connector.adoc

```diff
@@ -83,6 +83,7 @@ image::globaltype.jpg[GlobalType]
 |Field |Description
 |*Access Key* |Alphanumeric text string that uniquely identifies the user who owns the account. _Required_
 |*Secret Key* |Key that plays the role of a password. _Required_
+|*Security Token* |Temporary security credentials which are used to access the AWS resources. 
```

Comment:
Temporary security credentials used to access AWS resources.

Reply:
Updated